### ISSUE TITLE
Unsaon mani dili mani mo gana?

### ISSUE THREAD

**Issue by redwolfgang24** [redwolfgang24/sample-app-tdd#1]
Why this is not working?

**Comment by socialMedia18** [comment]
Hey Sir! I'm one of the students who attended the seminar in DCST building. I just want to ask if you have any tutorial about grails.(Groovy in Rails) because i am very  interested about that!

**Comment by redwolfgang24** [comment]
Nice, I'm glad that you are interested on grails. actually that's groovy on grails.

**Comment by redwolfgang24** [comment]
Yes, there's a lot pdf on how to learn about grails. This code that I share with you guys are the advance one so maybe you should know the basic stuff.

**Comment by redwolfgang24** [comment]
@socialMedia18 you have to install grails 2.4.5 first and the oracle java. If you are a linux user I suggest you should use sdkman to install grails, by that you can easily switch grails version if you wanted to upgrade into much higher version.

**Comment by redwolfgang24** [comment]
If you know me on facebook you can easily message me anytime, then if I have a time I would love to help. :)

**Comment by socialMedia18** [comment]
sorry for a late reply,  i'm just starting  learning about it, besides of being too busy of my thesis, i'll ask you later if i have a problem sir about groovy, so far im just reading a pdf about groovy. Thank you for giving a reply.
BTW sir! What is your fb account.

**Comment by redwolfgang24** [comment]
I'm glad to hear that :)... just send me an email, this repo is on public so I can't broadcast my fb account here.

**Comment by socialMedia18** [comment]
meandyou072013@yahoo.com

**Comment by redwolfgang24** [comment]
Good, May I know your name first?

**Comment by socialMedia18** [comment]
Not here Sir... :-)

**Comment by redwolfgang24** [comment]
Ok, I sent an email. please check your email.

**Comment by socialMedia18** [comment]
Ok, thank you Sir! I've got it!
